File: Perl/FileHandling.pl
open(FILE,  ">>test.txt") or die "COULD NOT OPEN THE FILE\n";

say FILE "The text was appended to the file";

open(FILE,  "<test.txt") or die "COULD NOT OPEN THE FILE\n";
while(<FILE>) {
   print "$_";
}
close FILE;

File: Python/FileHandling.py
f = open("Sample.txt", "r")

content = f.read()
sArr = content.split(' ')

mDict = {}

print(sArr)

for w in sArr:
    if w in mDict:
        mDict[w] += 1
    else:
        mDict[w] = 1

print(mDict)
f.close()